### DISCUSSION TITLE
Suggest edits to the drug list dictionary

### DISCUSSION THREAD

**Discussion by nabarunDG** [opioiddatalab/drugchecking#1] [Announcements]
Suggest new entries or modifications to the [street drug dictionary](https://github.com/opioiddatalab/drugchecking/blob/main/chemdictionary/chemdictionary.csv).

**Comment by kellybell** [comment]
Suggested edit to the drug list: Xylazine is listed twice, once early on and again at the end. I assume that's an error. So grateful for your work!! Thank you!
